plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -19,13 +19,18 @@
  * @return array{name: string, slug: string, short_description: string, requires: string|false, requires_php: string|false, requires_plugins: string[], download_link: string, version: string}|WP_Error Array of plugin data or WP_Error if failed.
  */
 function perflab_query_plugin_info( string $plugin_slug ) {
-	$transient_key = 'perflab_plugins_info-v2';
+	$transient_key = 'perflab_plugins_info';
```

Comment:
This was changed in https://github.com/WordPress/performance/pull/1613 to account for the format change. Since the transient only lasts for 1 day anyway, we can now restore the original transient key.